6.0/Dockerfile

```
@@ -25,7 +27,7 @@ RUN apt-get update \
 # Future Developments may include a "Slimmed" FSL 6.0 docker image
 RUN wget -P /tmp/ https://fsl.fmrib.ox.ac.uk/fsldownloads/fslinstaller.py \ 
 &&  mkdir -p /usr/share/fsl/ \ 
-&& python /tmp/fslinstaller.py -d /usr/share/fsl/6.0 -q
+&& python3 /tmp/fslinstaller.py -d /usr/share/fsl/6.0
```

The installer installs everything. However, there are a few things that for sure we're not going to need. For example, `fslview` and `FSLEYES`, since we're going to run headless.

```suggestion
&& python3 /tmp/fslinstaller.py -d /usr/share/fsl/6.0 \
&& rm -fr /usr/share/fsl/6.0/bin/fslview* \
    /usr/share/fsl/6.0/bin/FSLeyes \
    /usr/share/fsl/6.0/bin/fsleyes \
    /usr/share/fsl/6.0/src \
    /usr/share/fsl/6.0/doc \
    /usr/share/fsl/6.0/extras/src
```

---

Modified installer list to make image lighter.